src/main/java/com/google/sps/data/Post.java

```diff
@@ -61,6 +61,7 @@ public void requestToPost(HttpServletRequest request, String collegeId) {
     String numberOfPeopleItFeedsUnparsed = request.getParameter("numberOfPeopleItFeeds");
     String typeOfFoodUnparsed = request.getParameter("typeOfFood");
     String descriptionUnparsed = request.getParameter("description");
+    String collegeIdUnparsed = request.getParameter("collegeId");
```

Comment:
I couldn't find a change associated with this PR adding `collegeId` to the list of arguments provided in the JS POST - did it already do that previously?

Reply:
Yes, it was sent as a query parameter.